src/main.js

```diff
@@ -1,16 +1,17 @@
 const { extname } = require('path')
 
 require('./utils/polyfills')
-const { getFunctionInfos } = require('./info')
 const { getPluginsModulesPath } = require('./node_dependencies')
 const runtimes = require('./runtimes')
```

Comment:
[sand] Should the new `getFunctionsFromPaths()` be added to a separate file from the one importing each individual runtime type? Those look somewhat different. Also, this would also keeping `runtimes/index.js` as a "map" of languages/runtimes.